src/components/app/app.scss

```diff
@@ -1,6 +1,6 @@
 .app {
   display: flex;
-  height: 100vh;
+  height: 101vh;
```

Comment:
why are you using `101vh` ? it is not right

Reply:
just found a problem:
because of height: 100vh content of our page goes out of it. when I disable it, container stretches according to it's content.
is it possible not to use height with depending of window screen size?

with 100vh
![image](https://user-images.githubusercontent.com/14272396/37451196-89825a94-2839-11e8-9c98-da79bba4f452.png)
![image](https://user-images.githubusercontent.com/14272396/37451374-ffe758a6-2839-11e8-9c9c-93f48586f2f5.png)
this causes every time when content becomes higher than screen 

without 100vh
![image](https://user-images.githubusercontent.com/14272396/37451225-9e465660-2839-11e8-9f12-9667f47b60ee.png)